active_learning/tool_tracking_al/README.md

```diff
@@ -0,0 +1,50 @@
+# Active Learning Simulation Tutorial For Surgical Tool Segmentation
+
+Active learning (AL) is a machine learning framework that can serve as a data selection engine given an unlabeled pool of data. Active learning provides uncertainty estimation capabilities for trained deep learning models, in brief the models are able to assess about which predictions they are under or over confident. Based on the confidence the most relevant data for annotation can be selected. Thereafter annotation the labeled data is added to training pool of data for the model.
+
+This tutorial provides a toy example of active learning for surgical tool task segmentation. Exploratory notebooks on how to visualize uncertainty/confidence of model as provided as well
+
+### Steps to run the tutorial
+1. Download the data from [here](https://www.synapse.org/#!Synapse:syn22427422)
+2. To just launch the active learning script with 'variance' acquisition function, launch the code with the below command: `python active_learning.py --base_dir /to/be/defined --data_root /to/be/defined --json_path /to/be/defined --seed 120 --active_iters 6 --dropout_ratio 0.2 --mc_number 10 --initial_pool 15 --queries 15 --strategy variance --steps 2000 --val_interval 1 --batch_size 1 --val_batch_size 1 --lr 1e-4`
+3. For executing the active learning acquisition functions of `random`, `variance` and the `full dataset` baseline, execute the `./runner.sh`
```

Comment:
I am new to "active learning acquisition functions". Maybe some explanations can be helpful here. Thanks!

Reply:
I added some explanation